Description: remove some dependencies to be able to generate the doc package.
Forwarded: not-needed
--- a/setup.py
+++ b/setup.py
@@ -14,11 +14,7 @@
 
 install_requires=[
     # order is important
-    "setuptools",
-    "lxml",
     "vigilo-common",
-    "vigilo-models",
-    "networkx",
     "netifaces",
     ]
 if tuple(python_version_tuple()) < ('2', '6'):
